easybuild/easyconfigs/i/intel/intel-2019b.eb

```diff
@@ -0,0 +1,23 @@
+easyblock = 'Toolchain'
+
+name = 'intel'
+version = '2019b'
+
+homepage = 'https://easybuild.readthedocs.io/en/master/Common-toolchains.html#intel-toolchain'
+description = "Compiler toolchain including Intel compilers, Intel MPI and Intel Math Kernel Library (MKL)."
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+local_compver = '2019.4.227'
+local_gccver = '8.3.0'
+local_gccsuff = '-GCC-%s' % local_gccver
+dependencies = [
+    ('GCCcore', local_gccver),
+    ('binutils', '2.32', '-GCCcore-%s' % local_gccver),
+    ('icc', local_compver, local_gccsuff),
+    ('ifort', local_compver, local_gccsuff),
+    ('impi', '2019.4.243', '', ('iccifort', '%s%s' % (local_compver, local_gccsuff))),
```

Comment:
Use iimpi here?

Reply:
No, this must be `impi`, since the framework will verify the toolchain components (`icc`+`ifort` or `iccifort`, `impi`, `imkl`)